mLRS/Common/setup.h

```diff
@@ -153,6 +153,13 @@ void setup_configure_metadata(void)
     SetupMetaData.Tx_Buzzer_allowed_mask = 0; // not available, do not display
 #endif
 
+    // Tx Bridge WiFi Protocol : "TCP,UDP,BT"
+#ifdef DEVICE_BRIDGE_HAS_BT
```

Comment:
I'd like to suggest to rename this define to `DEVICE_HAS_ESP_WIFI_BRIDGE_ESP8266`
it would then have to be removed from the two stm32 tx hal files where you've added it, and be added to the respective esp tx hal files.
It then also needs to be added to the list of defines in hal.h for documentation.

otherwise, we could use a not "DEVICE_HAS_XXXXX" type define, like we do to specify the esp pins etc pp. Howverm, then it should be named starting with ESP_XXXX. Could be for instance ESP_IS_ESP8266. It when should be added not in the top however, but in the "esp" section down below, where also the ESP_ pin defines are.